tests/test_parser.py

```diff
@@ -6,7 +6,6 @@
 
 from vyxal.lexer import *
 from vyxal.parse import *
-from vyxal.structure import *
```

Comment:
I trust you @ysthakur that this is not going to horribly break everything and this PR is still approve-worthy